src/webview/webkitgtk/web_context.rs

```diff
@@ -221,14 +223,21 @@ where
     if let Some(uri) = request.uri() {
       let uri = uri.as_str();
 
-      //let headers = unsafe {
-      //  webkit_uri_request_get_http_headers(request.clone().to_glib_none().0)
-      //};
-
       // FIXME: Read the method
```

Comment:
we should be able to get the method now too, no? I think it was added in 2.36 too

Reply:
Yes, I'll add those too.